servicetest/testframework/lib.py

```diff
@@ -380,14 +380,22 @@ def __init__(self,
                  config_path=None,
                  caps_dir=None,
                  default_caps_dir=None,
-                 check_connection=True):
+                 check_connection=True,
+                 exec_prefix=None):
         if log_file_path is None:
             self.__log_file = subprocess.STDOUT
         else:
-            self.__log_file = open(log_file_path, "w")
+            if os.path.exists(log_file_path):
+                os.unlink(log_file_path)
```

Comment:
Is this really desirable? Don't we want to append on existing logs as well? Since the agent is class scoped, one could have the same logfile for several test classes in the same module, for example.